UnityProject/Assets/Scripts/Core/Transform/Orientation.cs

```diff
@@ -266,6 +266,7 @@ public override int GetHashCode()
 /// </summary>
 public enum OrientationEnum
 {
+	Default = -1,
```

Comment:
what does default represent?

Reply:
when opening a scene, lighttubes/lightbulbs that were looking to the right side were not having its updated since the OrientationEnum default is the right side already, so it was assumed they already had their sprite set